config/locales/doorkeeper.fr.yml

```diff
@@ -62,7 +62,7 @@ fr:
       buttons:
         revoke: Annuler
       confirmations:
-        revoke: Êtes-vous certain?
+        revoke: Êtes-vous certain ?
```

Comment:
This one is a good question. I personally also use a space, « Lexique de l'imprimerie nationale » also recommends a space, but other rules recommend nothing or ¼ quad space. And Belgian press for example uses nothing. Open source software tend to use space. The most important is to be coherent everywhere in the translation.